src/content/en/2022/privacy.md

```diff
@@ -1,22 +1,471 @@
+We find that the most prevalent library used on the web to perform fingerprinting is <a hreflang="en" href="https://github.com/fingerprintjs/fingerprintjs">FingerprintJS</a>, which we find on 0.62% of all websites (in our analysis, we looked for five different, known fingerprinting libraries). Most likely this is because the library is open source, and has a free version. Compared to [our measurements last year](../2021/privacy), we find that the use of fingerprinting has approximately stayed the same.
+
+### Evasion technique: CNAME tracking
+
+As most of the tracking countermeasures focus on blocking or disabling third-party cookies, another way to circumvent these protections is to use first-party cookies instead. Here, the tracker is cloaked using a CNAME record (hence the name) on a subdomain of the website it is embedded in. When the tracker then sets a cookie, it will be considered a first-party cookie.  A limitation of CNAME-based tracking is that it can only be used to track a user's activities within a specific website, although the tracker could still rely on <a hreflang="en" href="https://adtechexplained.com/cookie-syncing-explained/">cookie syncing</a> to match visits across multiple sites together.
+
+{{ figure_markup(
+  image="cname-tracking-services.png",
+  caption="Top 10 CNAME tracking services.",
+  description="Bar chart showing the prevalence of CNAME trackers. Adobe Experience Cloud was found on 0.65% desktop and 0.38% mobile sites respectively, Pardot on 0.44% and 0.25%, Oracle Eloqua on 0.06% and 0.03%, Act-On Software on 0.05% and 0.03%, Webtrekk on 0.02% and 0.01%, and Eulerian on 0.01% and 0.01%. The remaining CNAME trackers are shown with 0.00% prevalence. These are AT Internet, Tracedock, Ingenious Technologies, Criteo, Wizaly, and Keyade.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRSnqWDZQ07x6NdnIix7JOfg5DqGcoTrrpwaKavCIvNFkIANncyMxahKaTYjLka3rsrcBmrRIURpnPo/pubchart?oid=118406352&format=interactive",
+  sheets_gid="1717363829",
+  sql_file="nb_sites_with_cname_tracking.sql"
+  )
+}}
+
+By analyzing the various CNAME trackers, we find that the market share is mainly concentrated around two main services: Adobe Experience Cloud (0.65% on desktop and 0.38% on mobile) and Pardot (0.25% on desktop and 0.44% on mobile). Interestingly, the adoption of CNAME tracking is significantly higher on websites visited with a desktop browser compared to those visited on mobile. Presumably this is because there are fewer privacy-preserving mechanisms on mobile browsers (e.g. most of the popular browsers on mobile do not support extensions).
```

Comment:
```suggestion
By analyzing the various CNAME trackers, we find that the market share is mainly concentrated around two main services: Adobe Experience Cloud (0.65% on desktop and 0.38% on mobile) and Pardot (0.25% on desktop and 0.44% on mobile). Interestingly, the adoption of CNAME tracking is significantly higher on websites visited with a desktop browser compared to those visited on mobile. Presumably, this is because there are fewer privacy-preserving mechanisms on mobile browsers (e.g. most of the popular browsers on mobile do not support extensions).
```